include/kokkos-utils/callbacks/Events.hpp

```diff
@@ -261,6 +261,12 @@ concept NamedEvent =
         { event.name } -> std::same_as<std::string&>;
     };
 
+//! Concept to constrain any event type that has a member variable @c event_id.
+template <typename EventType>
+concept IdentifiableEvent = Event<EventType> && requires (EventType& event) {
```

Comment:
Should we do actually `IndexedEvent`? To be consistent with `NamedEvent` and `EnqueuedEvent`?

Reply:
:+1: